cognite/neat/graph/loaders/_base.py

```diff
@@ -65,11 +65,11 @@ def load_into_cdf_iterable(self, client: CogniteClient, dry_run: bool = False) -
                 items.append(result)
 
             if len(items) >= self._UPLOAD_BATCH_SIZE:
-                yield self._upload_to_cdf(client, items, dry_run, issues)
+                yield from self._upload_to_cdf(client, items, dry_run, issues)
                 issues = NeatIssueList[NeatIssue]()
                 items = []
         if items:
-            yield self._upload_to_cdf(client, items, dry_run, issues)
+            yield from self._upload_to_cdf(client, items, dry_run, issues)
 
     @abstractmethod
     def _get_required_capabilities(self) -> list[Capability]:
```

Comment:
considering issue that we bump into when client IAM did not have any cap. do we have a fall back when loader will attempt to push things to CDF?

Reply:
Not sure what you mean here.